templates/index.html

```diff
@@ -199,7 +198,7 @@
     <a href='.' class="custom-navtitle">
       <strong>Meilix Generator</strong>
     </a>
-    <div class="custom-menubutton">
+    <div onclick="document.getElementsByClassName('custom-menu-cont')[0].classList.toggle('hidden')" class="custom-menubutton">
```

Comment:
You could put this into a function e.g. "toggleMenu" to express the intent. The functionality is clear.